src/rules/order.js

```diff
@@ -609,18 +614,18 @@ module.exports = {
       },
       TSImportEqualsDeclaration: function handleImports(node) {
         let name
+        let type = 'import'
         if (node.moduleReference.type === 'TSExternalModuleReference') {
           name = node.moduleReference.expression.value
-        } else if (node.isExport) {
-          name = node.moduleReference.name
         } else {
-          name = null
+          name = context.getSourceCode().getText(node.moduleReference)
+          type += ':object'
```

Comment:
```suggestion
          type = 'import:object'
```
this makes it easier to grep for

Reply:
Will do